_i18n/de/resources/user-guides/ledger-wallet-cli.md

```diff
@@ -1,165 +1,167 @@
 
-4. Your Ledger needs to be plugged in and the Ledger Monero app should be running.
+4. Während die Ledger-App von Monero bereits läuft, muss nun dein Ledger angeschlossen werden.
 
-5. Either your daemon (`monerod.exe`) should be running and preferably be fully synced or you should connect to a remote node.
+5. Du solltest entweder deinen (vollständig synchronisierten) Hintergrunddienst (`monerod.exe`) betreiben oder eine Verbindung zu einem Remote-Node herstellen.
 
-Now that we're sufficiently prepared, let's start!
+Jetzt, da wir ausreichend vorbereitet sind, können wir beginnen!
 
-1. Go to the directory / folder monerod.exe and monero-wallet-cli.exe are located.
+1. Gehe in das Verzeichnis oder in den Ordner, in welchem monerod.exe und monero-wallet-cli.exe abgespeichert sind.
 
-2. Open a new command prompt / powershell. This is done by first making sure your cursor isn't located on any of the files and subsequently doing SHIFT + right click. It will give you an option to "Open command window here". If you're using Windows 10 in latest version, it'll give you an option to "open the PowerShell window here".
+2. Öffne eine neue Befehlszeile oder PowerShell. Dies kannst du tun, indem du zunächst sicherstellst, dass dein Cursor nicht auf einer der Dateien liegt, und dann mit der gedrückten Umschalttaste rechtsklickst. Dir wird nun die Option "Befehlszeile hier öffnen" angezeigt. Die letzte Version von Windows 10 bietet außerdem die Option "PowerShell-Fenster hier öffnen".
 
-3. Now type:
+3. Jetzt gibst du Folgendes ein:
 
 `monero-wallet-cli.exe --generate-from-device <new-wallet-name> --subaddress-lookahead 3:200` (Win 7 + 8)
 
 `.\monero-wallet-cli.exe --generate-from-device <new-wallet-name> --subaddress-lookahead 3:200` (Win 10)
 
-Note that is simply a placeholder for the actual wallet name. If you, for instance, want to name your wallet `MoneroWallet`, the command would be as follows:
+Beachte, dass `<new-wallet-name>` schlicht der Platzhalter des tatsächlichen Namens deines Wallets ist. Wenn du dein Wallet beispielsweise `MoneroWallet` nennen möchtest, würde der Befehl folgendermaßen aussehen:
 
 `monero-wallet-cli.exe --generate-from-device MoneroWallet --subaddress-lookahead 3:200` (Win 7 + 8)
 
 `.\monero-wallet-cli.exe --generate-from-device MoneroWallet --subaddress-lookahead 3:200` (Win 10)
 
-4. The CLI will, after executing aforementioned command, prompt your for a password. Make sure to set a strong password and confirm it thereafter.
+4. Nachdem der vorige Befehl ausgeführt wurde, wird CLI dich zur Passworteingabe auffordern. Stelle sicher, dass du ein starkes Passwort verwendest und dieses anschließend bestätigst.
```

Comment:
@stmax82 Just to understand completely: it would be okay for you to keep the abbrevation, but with a prefixed article (you used "die CLI") like you suggested? In that case I'd just switch to "das" (like "das CLI-Wallet"; in this case it even fits with "das Interface").

Reply:
Abbrevation should be fine (like in English), but it needs an article. I think you used "das" several times in other places, so I'd use "das CLI" here too.